score/prepareTestFiles.ts

```diff
@@ -83,13 +92,15 @@ function createTestFile(content: string, fileName: string) {
 
 const createTestInfoFile = (testData: TestRequiredData, path: string) => {
   const { id, pluginVersion, vsCodeVersion } = testData;
-  const testInfo = `
-    minionTaskId: ${id}
-    pluginVersion: ${pluginVersion}
-    vsCodeVersion: ${vsCodeVersion}
-    date of test: ${Date.now().toLocaleString()}
+  const testInfo = `{
+    "minionTaskId": "${id}",
+    "pluginVersion": "${pluginVersion}",
+    "vsCodeVersion": "${vsCodeVersion}",
+    "date": "${getTodayDate()}", 
+    "score": "null"
+  }
```

Comment:
Next time you can operate on regular objects and stringify automatically
```tsx
const testInfo = JSON.stringify({
  minionTaskId: id,
  score: null,
}, null, 2);

Reply:
good point, will replace it ;)